src/androidTest/java/com/mixpanel/android/mpmetrics/FCMMessagingServiceTest.java

```diff
@@ -91,6 +94,14 @@ private void verifyBasicNotification(Notification.Builder builder, long now) {
         verify(builder).setLargeIcon(any(Bitmap.class));
     }
 
+    private void verifyLargeIcon(Notification.Builder builder) {
+        verify(builder).setLargeIcon(any(Bitmap.class));
```

Comment:
looks like we already call this as a part of the `verifyBaseNotification` method. I think what we want there is to have a separate test where we test both variations: a "resource name" and a "url"